tests/model_formsets/tests.py

```diff
@@ -1267,14 +1267,66 @@ def test_modelformset_validate_max_flag(self):
         FormSet = modelformset_factory(Price, fields="__all__", extra=1, max_num=1, validate_max=True)
         formset = FormSet(data)
         self.assertFalse(formset.is_valid())
-        self.assertEqual(formset.non_form_errors(), ['Please submit 1 or fewer forms.'])
+        self.assertEqual(formset.non_form_errors(), ['Please submit at most 1 forms.'])
 
         # Now test the same thing without the validate_max flag to ensure
         # default behavior is unchanged
         FormSet = modelformset_factory(Price, fields="__all__", extra=1, max_num=1)
         formset = FormSet(data)
         self.assertTrue(formset.is_valid())
 
+    def test_modelformset_min_num_equals_max_num_less_than(self):
+        # If validate_max and validate_min is set and max_num is less than
+        # TOTAL_FORMS in the data, then this throws an exception.
+
+        data = {
+            'form-TOTAL_FORMS': '3',
+            'form-INITIAL_FORMS': '0',
+            'form-MAX_NUM_FORMS': '2',  # should be ignored
```

Comment:
We can chop this comment, it's not related with tested behavior (the same in the second test).